R/derive_param_computed.R

```diff
@@ -468,13 +537,13 @@ get_hori_data <- function(dataset,
   )
 
   # horizontalize data, e.g., AVAL for PARAMCD = "PARAMx" -> AVAL.PARAMx
-  analysis_vars <- extract_vars(analysis_value)
+  analysis_vars <- flatten(map(unname(set_values_to), extract_vars))
```

Comment:
`flatten()` was superseded in purrr 1.0.0, not sure if we need to watch out for that

Reply:
Yes, we should change to `list_flatten()` in future. We can't do it now because it would break the R 4.1 check.